build/generate-style-spec.ts

```diff
@@ -38,6 +38,8 @@ function propertyType(property) {
                 return 'SkySpecification';
             case 'sources':
                 return '{[_: string]: SourceSpecification}';
+            case 'projection:':
+                return 'ProjectionSpecification';
```

Comment:
I think you need to define this somewhere so that it can be added to the typescript definitions, we did a similar thing for padding, I think you can try and copy from it.

Reply:
I ran generate-typings, and I think it might be working already.

export type ProjectionSpecification = {
	"type"?: PropertyValueSpecification<"mercator" | "globe">;
};

<img width="853" alt="Screenshot 2024-11-05 at 08 57 18" src="https://github.com/user-attachments/assets/61541f35-34c4-4d78-b3ed-14a48d22c5be">